net/HttpRequest.hpp

```diff
@@ -755,9 +755,9 @@ class Request final
     void dumpState(std::ostream& os, const std::string& indent = "\n  ") const
     {
         os << indent << "http::Request: " << _version << ' ' << _verb << ' ' << _url;
-        os << indent << "stage: " << name(_stage);
-        os << indent << "headers: ";
-        _header.serialize(os);
+        os << indent << "\tstage: " << name(_stage);
```

Comment:
Curious as to why we're adding another indent to indent in-line here, rather than just bumping that in the parent caller ?

Reply:
It makes it more readable to have the indents:
```
http::server::Session: #123 
    connected: ...
    startTime: ...
```

Rather than have them all on the same level. It's just a matter of indenting the members of the object vs it's title, for readability (these dumps do get very long and tiring/confusing to read).

Reply:
Not a fan of the large number of lines changed; but hey.